vertx-vaadin-flow-parent/vertx-vaadin-flow/README.md

```diff
@@ -21,29 +18,26 @@ All [Servlet Configuration Parameter](https://vaadin.com/docs/-/part/framework/a
 }
 ``` 
 
-Two dependencies are needed to run Vaadin on Vert.x: `vertx-vaadin` adapter and servlet APIs (because Vaadin relies on them
-but they are not provided from Vert.x).
+Two dependencies are needed to run Vaadin on Vert.x: `vertx-vaadin` adapter and Servlet APIs (because Vaadin relies on them but they are not provided by Vert.x).
 
 ```xml
 <dependency>
   <groupId>com.github.mcollovati.vertx</groupId>
   <artifactId>vertx-vaadin-flow</artifactId>
-  <version>0.2.0</version>
+  <version>13.0.2</version>
 </dependency>
 
 <dependency>
-    <groupId>javax.servlet</groupId>
-    <artifactId>javax.servlet-api</artifactId>
-    <version>${javax.servlet-api.version}</version>
+    <groupId>jakarta.servlet</groupId>
+    <artifactId>jakarta.servlet-api</artifactId>
+    <version>4.0.2</version>
```

Comment:
Vaadin Flow currently depends on `javax.servlet:javax.servlet-api:jar:3.1.0`. What do you think if we suggest using the same version instead of the latest one?

Reply:
I think as a general rule, the more recent a dependency, the better. And moving to Jakarta would be a good idea (but they don't have an older version). But, if Vaadin hasn't moved, and you want to stay in sync with them, then I'm fine with you altering this PR.

Reply:
I noticed you also proposed on Gitter a PR to update servlet dependendy on Vaadin Flow.
I think we can go with jakarta dependency since in packages has not yet changed and also Vaadin will bump dependency soon or later